login/handler.go

```diff
@@ -192,9 +205,9 @@ func (h *Handler) handleRefresh(w http.ResponseWriter, r *http.Request, userInfo
 	}
 }
 
-func (h *Handler) deleteToken(w http.ResponseWriter) {
+func (h *Handler) deleteCookie(w http.ResponseWriter, cookieName string) {
```

Comment:
When `h.config.CookieDomain` is set, this will not remove the `h.config.RedirectQueryParameter` cookie - as this does not have a `Domain` set.

Thus, I am on "sub1.domain.com" with "domain.com" set as the Cookie-Domain, the `backTo` cookie is valid on "sub1.domain.com" only (not an issue), but the cookie delete is explicitly set to the "domain.com" domain.

Either set the `backTo` to be on same cookie domain (will cause issues if the cookie is left over and you hit a different sub-domain) or add some other logic here to set the domain when needed.

Reply:
Okay I think I've fixed this by just deleting the cookie the same way I set it rather than reusing the deleteToken function.  I've returned the deleteToken function back to what it was before.  Did I miss something?